Console/Command/RestoreUseDefaultValueCommand.php

```diff
@@ -52,6 +67,8 @@ public function execute(InputInterface $input, OutputInterface $output)
         $counts = array();
         $i = 0;
         $tables = array('varchar', 'int', 'decimal', 'text', 'datetime');
+        $column = $this->productMetaData->getEdition() === 'Enterprise' ? 'row_id' : 'entity_id';
```

Comment:
I do not have access to the Enterprise edition. Could anyone confirm this?

Reply:
This is where I got the original code from: https://magento.stackexchange.com/questions/134265/magento-2-programmatically-check-magento-version-ce-ee

Reply:
I meant if it is really called `row_id` in the Enterprise edition. But I just believe you :)